tests/superuser-secret/superuser_secret_suite_test.go

```diff
@@ -1,132 +0,0 @@
-// Copyright DataStax, Inc.
-// Please see the included license file for details.
-
-package superuser_secret
-
-import (
-	"fmt"
-	"testing"
-
-	"encoding/base64"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
-	ginkgo_util "github.com/datastax/cass-operator/mage/ginkgo"
-	"github.com/datastax/cass-operator/mage/kubectl"
-)
-
-var (
-	testName          = "Superuser Secret"
-	namespace         = "test-superuser-secret"
```

Comment:
did this get duplicated into one of the above two tests?

Reply:
Right, I should have made a comment about that. This was replaced by the other two superuser secret tests.